### PR TITLE
Improve documentation for Docker

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Make it as simple and accessible as I can.

Also, do several tweaks here and there.

Closes FerretDB/engineering#27.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [x] I added/updated comments and checked rendering.
* [x] I made spot refactorings.
* [x] I updated user documentation.
* [x] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: build/version/generate.go [hunk 1]
```diff
@@ -76,10 +76,7 @@ func main() {
 	}()
 
 	// output package.txt in the same format just for logging
-	b, err := os.ReadFile("package.txt")
-	if err != nil {
-		b = []byte(err.Error())
```
Comment (AlekSi) [inline build/version/generate.go:81]: That error message was confusing

File: build/version/version.go [hunk 2]
```diff
@@ -43,8 +43,10 @@
 //   - The default logging level is set to debug.
 //
 // Debug builds are orthogonal to production releases, development releases, and local/host builds.
-// For example, the host build could be made non-debug, and the production release could be a debug build,
-// although that's not common.
+// For example, the host build could be made non-debug, and the production release, in theory, could be a debug build.
+//
+// Currently, our production releases are non-debug builds,
+// and all other builds and releases (development releases, all-in-one releases, local builds, etc.) are debug builds.
```
Comment (AlekSi) [inline build/version/version.go:49]: It is somewhat hard to understand, but almost no-one will read it anyway

File: website/docs/configuration/logging.md [hunk 3]
```diff
@@ -9,15 +9,26 @@ description: Logging
 
 Logs from FerretDB running on Docker can be accessed through the container.
 
-If Docker was launched with the [quick start](../quickstart-guide/docker.md#setup-with-docker-compose),
+If Docker was launched with [our quick local setup with Docker Compose](../quickstart-guide/docker.md#setup-with-docker-compose),
 the following command can be used to fetch the logs.
 
-```shell
-docker logs -name ferretdb-docker-ferretdb-1
```
Comment (AlekSi) [inline website/docs/configuration/logging.md:16]: That command wasn't correct – the container name depends on the directory name where docker-compose.yml was stored

File: website/docs/configuration/logging.md [hunk 4]
```diff
@@ -9,15 +9,26 @@ description: Logging
...
+CONTAINER ID   IMAGE                       COMMAND                  CREATED              STATUS          PORTS                                           NAMES
+13db4c8800d3   postgres                    "docker-entrypoint.s…"   About a minute ago   Up 59 seconds   5432/tcp                                        my-postgres
+44fe6f4c3527   ghcr.io/ferretdb/ferretdb   "/ferretdb"              About a minute ago   Up 59 seconds   8080/tcp, 27018/tcp, 0.0.0.0:27017->27017/tcp   my-ferretdb
+
+$ docker logs my-ferretdb
 ```
 
 ## Binary executable logs
 
-FerretDB generates logs to standard output `stdout` and standard error `stderr` streams
```
Comment (AlekSi) [inline website/docs/configuration/logging.md:21]: We don't log to stdout

File: website/docs/quickstart-guide/docker.md [hunk 5]
```diff
@@ -6,23 +6,23 @@ description: How to set up FerretDB using Docker
...
+The rest are covered below.
 
-* [Production image](https://ghcr.io/ferretdb/ferretdb) `ghcr.io/ferretdb/ferretdb`.
-* [Development image](https://ghcr.io/ferretdb/ferretdb-dev) `ghcr.io/ferretdb/ferretdb-dev`.
-* [All-in-one image](https://ghcr.io/ferretdb/all-in-one) `ghcr.io/ferretdb/all-in-one`.
+## Production image
 
-The last one is provided for quick testing and experiments and is unsuitable for production use cases.
-It is documented in the [FerretDB repository](https://github.com/FerretDB/FerretDB#quickstart).
+Our [production image](https://ghcr.io/ferretdb/ferretdb) `ghcr.io/ferretdb/ferretdb`
```
Comment (AlekSi) [inline website/docs/quickstart-guide/docker.md:20]: That's the only `ghcr.io` text on that page now

File: website/docs/quickstart-guide/docker.md [hunk 6]
```diff
@@ -6,23 +6,23 @@ description: How to set up FerretDB using Docker
 
 # Docker
 
-## Provided images
+You provide three Docker images for various deployments:
```
Comment (rumyantseva) [inline website/docs/quickstart-guide/docker.md:9]:
```suggestion
We provide three Docker images for various deployments:
```

I guess

File: website/docs/quickstart-guide/docker.md [hunk 7]
```diff
@@ -6,23 +6,23 @@ description: How to set up FerretDB using Docker
...
+"all-in-one" for quick testing and experiments,
+a development image for debugging problems,
+and a production image for all other cases.
```
Comment (rumyantseva) [inline website/docs/quickstart-guide/docker.md:12]: If we don't make them a list, let's make them bold?

```suggestion
**"all-in-one"** for quick testing and experiments,
**a development image** for debugging problems,
and **a production image** for all other cases.
```

**Review by rumyantseva — APPROVED** [review]
LGTM, left a couple of suggestions, feel free to decline them if disagree.

File: website/docs/quickstart-guide/docker.md (hunk 6 above)
Comment (AlekSi) [reply website/docs/quickstart-guide/docker.md]: Doh!

File: .github/workflows/docs-preview.yml [hunk 8]
```diff
@@ -35,7 +35,7 @@ jobs:
 
     environment:
       name: cloudflare-dev
-      # url: ${{ steps.extract.outputs.url }} TODO https://github.com/FerretDB/github-actions/issues/160
+      url: ${{ steps.extract.outputs.deployment_url || steps.extract.outputs.extracted_url}}
```
Comment (AlekSi) [inline .github/workflows/docs-preview.yml:38]: https://github.com/FerretDB/github-actions/pull/165

**Review by KrishnaSindhur — DISMISSED** [review]

**Review by ptrfarkas — APPROVED** [review]

**Pull request merged**